universe/interface.go

```diff
@@ -360,11 +360,12 @@ type Syncer interface {
 // DiffEngine is a Universe diff engine that can be used to compare the state
 // of two universes and find the set of assets that are different between them.
 type DiffEngine interface {
-	BaseForest
```

Comment:
Was mainly done to dedup on method names. Which other methods would be added here?

Reply:
I've added two more commits to this PR. They add a `RegisterIssuance` method to the base forest. When called this new method will update the base universe and the universe forest DB tables using a single DB transaction.

I don't think `DiffEngine` should include that sort of write methods.